core/drivers/atmel_tcb.c

```diff
@@ -50,7 +50,12 @@
 #define TCB_WPMR		0xe4
 #define  TCB_WPMR_WAKEY		0x54494d
 
+#ifdef CFG_SAMA7G5
+static const char * const tcb_clocks[] = {"t0_clk", "t1_clk", "t2_clk",
+					  "slow_clk"};
```

Comment:
nitpicking: could you add space chars after `{` and before `}`.
suggestion:
```c
static const char * const tcb_clocks[] = {
	"t0_clk", "t1_clk", "t2_clk", "slow_clk"
};

Reply:
Updated and added the review tags. Thanks.